lib/common/xml.c

```diff
@@ -1021,37 +1021,49 @@ pcmk__strip_xml_text(xmlNode *xml)
     }
 }
 
+/*!
+ * \internal
+ * \brief Parse XML from a file
+ *
+ * \param[in] filename  Name of file containing XML (\c NULL or \c "-" for
+ *                      \c stdin); if \p filename ends in \c ".bz2", the file
+ *                      will be decompressed using \c bzip2
+ *
+ * \return XML tree parsed from the given file; may be \c NULL or only partial
+ *         on error
+ */
 xmlNode *
-filename2xml(const char *filename)
+pcmk__xml_parse_file(const char *filename)
```

Comment:
That does seem clearer. I was trying to stick with the `parse` convention since we're from text to a structured representation (XML). But it's hard to imagine reading XML *without* doing that.

Reply:
Done